### ISSUE TITLE
Class File Name Ruleset Showing Incorrect Error

### ISSUE THREAD

**Issue by kopepasah** [AtomLinter/linter-phpcs#248]
The PHPCS linter for Atom is displaying an Error for class + filename, when no error actually exists.

This is the error I receive with `class Menus` and `class-menus.php`:

```php
Error	PHPCS	[WordPress.Files.FileName.InvalidClassFileName] Class file names should be based on the class name with "class-" prepended. Expected class-menus.php, but found class-menus.php".	1:1
```

Notice that the expected file name and found file name is the same.

After debugging, I have found that running `phpcs` from shell does not produce this error, nor does this error exist in other IDEs.

---

Linter (this repo): 1.6.0
Atom: 1.15.0
PHPCS: 2.8.1
WPCS: 0.11.0

**Comment by synthetiv** [comment]
I've been having this issue too, and I had a hunch when I noticed the trailing quote in the filename (`class-menus.php"` in the example above). I've submitted a PR and hope to add tests for this in the next week or so.

**Comment by kopepasah** [comment]
Actually, I found this was a direct issue with WPCS and not with this linter. A pull request was merged to trim the quotes:

https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/pull/928
